model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntityToSampleManifest.java

```diff
@@ -31,12 +31,12 @@ public void setSampleManifestEntity(SampleManifestEntity sampleManifestEntity) {
         this.sampleManifestEntity = sampleManifestEntity;
     }
 
-    public SampleManifest getSampleManifest() {
-        return sampleManifest;
+    public SampleManifestJsonEntity getSampleManifest() {
```

Comment:
the getter method name should match the field it corresponds to (`getSampleManifestJsonEntity`)